src/server/scripts/Outland/zone_shadowmoon_valley.cpp

```diff
@@ -751,7 +751,7 @@ class npcs_flanis_swiftwing_and_kagrosh : public CreatureScript
             uint8 msg = player->CanStoreNewItem(NULL_BAG, NULL_SLOT, dest, 30658, 1, NULL);
             if (msg == EQUIP_ERR_OK)
             {
-                player->StoreNewItem(dest, 30658, 1, true);
```

Comment:
Was it an error ? Because `player->StoreNewItem(ItemPosCountVec, uint32, int32, bool)` doesn't exist